logging/src/lib.rs

```diff
@@ -57,5 +62,8 @@ pub fn setup_tracing() -> impl Drop {
         .expect("Failed to initialize tracing subscriber");
 
     info!("Tracing initialized successfully.");
+    trace!("Trace-level logging is enabled for block_processor.");
```

Comment:
same, this message references another module that it is independent of, remove

Reply:
And aslo here too.